src/js/configs.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
   // aws elb describe-load-balancer-policies --query "PolicyDescriptions[?PolicyName=='ELBSample-ELBDefaultCipherPolicy'].PolicyAttributeDescriptions[*].AttributeName[]"
   awselb: {
     hasVersions: false,
-    highlighter: 'json',
-    latestVersion: '2015.3',
+    highlighter: 'yaml',
+    latestVersion: '2014-02-19',
```

Comment:
It's not going to break things here, but I try to keep all the versions as semantic versions, e.g. 2014.2.19.

Reply:
Good idea, I'll change this.